Sentry.podspec

```
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.frameworks = 'Foundation'
   s.libraries = 'z', 'c++'
+  s.swift_versions = "5.5"
```

`h`: I think we should change this to
```suggestion
  s.swift_versions = ['5']
```

According to the [podspec](https://guides.cocoapods.org/syntax/podspec.html)

> The versions of Swift that the specification supports. A version of '4' will be treated as '4.0' by CocoaPods and not '4.1' or '4.2'.
> 
> Note The Swift compiler mostly accepts major versions and sometimes will honor minor versions. While CocoaPods allows specifying a minor or patch version it might not be honored fully by the Swift compiler.

So it mostly accepts major versions anyways, and why do we already want to jump to 5.5? Can't we still support 5.0?

Maybe 5.5 because it still works with Xcode 13.0, which still allows developing for iOS 9, which is our min iOS version? If yes, please put this info as a comment here.

---

It was your idea to support a version bigger than 5, thats why I changed. It made sense to me. 
Our CocoaPods supports 5.4, since this will be a major, why not ditch one minor. 5.5 is supported by Xcode 12, and this is the minimum Xcode we test anyway.

---

Could setting this to `5.5` break someone?

---

Only if Xcode don't support 5.5. 
Which means there is someone out there using Xcode 11. 
I don't believe this is a thing.